src/khoj/processor/conversation/prompts.py

```diff
@@ -432,7 +432,7 @@
 - Break messages into multiple search queries when required to retrieve the relevant information.
 - Use site: google search operators when appropriate
 - You have access to the the whole internet to retrieve information.
-- Official, up-to-date information about you, Khoj, is available at site:khoj.dev, github or pypi.
+- Official, up-to-date information about you, Khoj, is available at site:khoj.dev
```

Comment:
Please revert the changes in this file 🙂

Reply:
These changes weren't directly made by me. They seem to have been made when I pulled the files from master. I'll have them removed!

Reply:
You may need to pull from master again.